tests/Domain.UnitTests/Domain.UnitTests.csproj

```diff
@@ -11,13 +11,13 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
-        <PackageReference Include="nunit" Version="3.13.3" />
-        <PackageReference Include="NUnit3TestAdapter" Version="4.2.1">
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0-preview-23503-02" />
```

Comment:
previews, betas, alphas? 

no stable versions that work with .NET 7?

Reply:
That's a mistake - I'll roll that back

Reply:
Done.